packages/email/src/components/Head.mjml

```diff
@@ -0,0 +1,11 @@
+import mjml2html from 'mjml';
+
+const htmlOutput = mjml2html(`
+<mjml>
+  <mj-head>
+    <mj-font name="Raleway" href="https://fonts.googleapis.com/css?family=Raleway"/>
```

Comment:
Not a blocker but just a nitpick: I don't like that we ping Google servers here every time someone opens a mail. Is there a way to include the font?

If not, no big deal :)